### ISSUE TITLE
Search query expiry sorting

### ISSUE THREAD

**Issue by barnabycollins** [Kacper-Lubisz/TrayMaster#24]
When working on the `search` branch, trays returned from a search query are sorted correctly by the start date of their expiry window, but arbitrarily in terms of expiry window size. I think the best solution to this would be to have a secondary sort in ascending order of window size (that is, always order Jan 2020 before Q1 2020 before 2020) - that way, we're consistently prioritising trays with the soonest 'average' expiry date. For example, you might get:

- Jan 2020
- Q1 2020
- 2020
- Feb 2020
- Mar 2020
- Apr 2020
- Q2 2020
- Jan 2030
- 2030

TL;DR: I think when sorting by expiry we should sort first in order of ascending expiry window start and then in order of ascending window size.
